mobile/lib/modules/profile/widgets/profile/list_status_shimmer.widget.dart

```diff
@@ -16,62 +18,66 @@ class ListStatus extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return isLoading
-        ? Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: List.generate(3, (index) {
-              return Shimmer.fromColors(
-                baseColor: Colors.grey[300]!,
-                highlightColor: Colors.grey[100]!,
-                child: Column(
-                  children: [
-                    Container(
-                      width: 30,
-                      height: 35,
-                      decoration: BoxDecoration(
-                        color: Colors.white,
-                        borderRadius: BorderRadius.circular(5),
-                      ),
+    return BlocBuilder<AuthBloc, AuthState>(
+      builder: (context, state) {
+        return isLoading
+            ? Row(
```

Comment:
If widget is too long, I think you should extract to the method like _buildLoadingFooWidget() or something else for readability.

Reply:
👀